test/splitWith.js

```diff
@@ -0,0 +1,23 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+
+describe.only('splitWith', function() {
+  it('splits an array at every instance of predicate matching', function() {
+    eq(R.splitWith(R.equals(2), [1, 2, 3, 4, 2, 1]), [[1], [3, 4], [1]]);
+    eq(R.splitWith(R.equals('a'), 'bananas'), [['b'],['n'],['n'],['s']]);
```

Comment:
If this is the behavior, I'm not sure I want to promote it in a test case.  To my mind, if we accept Strings intentionally rather than as an accident of implementation, then this should really return `['b', 'n', 'n', 's']`  -- although I like the example and the cross-function consistency in this example.

Reply:
That makes sense, I changed it to an array of strings (which has the same output which highlights the issue)